### PR TITLE
test: tighter check for outputs

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area tests

**What this PR does / why we need it**:

This PR allows our integration test suite to perform a stricter check on Falco's outputs. 

That's particularly useful to ensure we are not breaking things while refactoring (e.g., https://github.com/falcosecurity/falco/pull/1412).

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/confs/stdout_output.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2019 The Falco Authors.
```
Comment (fntlnz) [inline test/confs/stdout_output.yaml:2]: 2020

File: test/confs/program_output.yaml [hunk 2]
```diff
@@ -41,4 +41,4 @@ stdout_output:
 
 program_output:
   enabled: true
-  program: cat > /tmp/falco_outputs/program_output.txt
+  program: cat >> /tmp/falco_outputs/program_output.txt
```
Comment (fntlnz) [inline test/confs/program_output.yaml:44]: This command keeps a process open, that's what you want?

Comment (leogr) [reply test/confs/program_output.yaml:44]: It's not really important, Falco will close the program :point_down: 

https://github.com/falcosecurity/falco/blob/8b56360f8ca2f088c27ac124b01dccdce8ec5bc6/userspace/falco/lua/output.lua#L136

**Review by leodido — APPROVED** [review]
Finally my beloved tests 😃 

Bravo!

Nitpicking: the automatic formatting of the Python code made it difficult to review it (falco_tests.py). I'm a fan of ^K+S in vscode 😸

**Comment by leodido** [comment]
@leogr do not forget to build the `falco-tester` container image and push it :)

**Comment by leogr** [comment]
> @leogr do not forget to build the `falco-tester` container image and push it :)

It should not be needed, this PR does not touch dependency or any other things that's contained in `falco-tester`.

Regarding the automatic formatting, I just noticed that when it was too late, sorry :smile_cat:

**Pull request merged**